### DISCUSSION TITLE
Checkout and Order items does not match

### DISCUSSION THREAD

**Discussion by zhaomengru2015** [Shopify/storefront-api-feedback#176] [Help]
I have three items in cart
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/9380107/221580106-08333b52-0526-4af9-8b99-9b6b2f0160cf.png">

"Keystone Wallet Pouch" is 50% off because we setup an automatic discount (buy 1 "Keystone Pro", get "Keystone Wallet Pouch" 50% off)

when I checkout, the items display as follows:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/9380107/221580952-9b2db084-f993-405c-b282-d991e67c608e.png">

Although the items counts  and prices are correct, they are a bit of inconsistent. But when I  create an order, the "Keystone Pro" are doubled.

<img width="670" alt="image" src="https://user-images.githubusercontent.com/9380107/221581727-1a6290e6-bdaa-45e1-a658-433905606dd3.png">

it's so weird. I am sure this is caused by the "automatic discount rule", because when I only order "Keystone Pro", everything works as expected.
Please have a look and get back to us soon.

**Comment by vixdug** [comment]
Thanks @zhaomengru2015 - I agree that this is caused by the discount rule. We will look into this!

**Comment by zhaomengru2015** [comment]
@vixdug Hi thanks for helping. More info here:  when I look into the order data, and I found out that for  all the "doubled items",  "fulfillable_quantity" are doubled compared with "quantity"
<img width="611" alt="image" src="https://user-images.githubusercontent.com/9380107/221787957-3936b38c-7dcc-4772-a99c-5c95508a9672.png">
Please have a look into it and get back to us, our business is heavily impacted.
